### DISCUSSION TITLE
General PR policy

### DISCUSSION THREAD

**Discussion by mhesselbarth** [iiasa/BNRTools#3] [General]
Hey all, 

We decided to work on our own branches (e.g., `max`) to avoid conflicts. Once done and tested, we can then merge into `dev` and every now and then update `main`. I was just wondering what the general approach to PR into `dev` are. Should we wait until somebody else had a quick review of a PR or just merge once done?

Related to this, can we push small fixes directly to `dev`, especially now at the beginning because there are probably a few minor changes needed to configure e.g. `DESCRIPTION` etc.?

**Comment by Martin-Jung** [comment]
Mhh, I think at the start we could just merge things in as we go (I just merged your PR :D ). 
But as soon as the package is regularly used in productive use, we might need a pair of eyes to avoid breaking changes. 
Maybe the rules here can be for cases when:
- Functions are renamed
- Parameters are renamed
- Output or inputs are changed
?

**Comment by mhesselbarth** [comment]
I think that's a good policy. So okay to just merge as long as none of the above cases apply. If they do, require review by someone
